cms/envs/common.py

```diff
@@ -1135,6 +1141,10 @@
 # TTL for microsite database template cache
 MICROSITE_DATABASE_TEMPLATE_CACHE_TTL = 5 * 60
 
+# Cache expiration for the version of the footer served
+# by the branding API.
+FOOTER_CACHE_TIMEOUT = 30 * 60
```

Comment:
This seemed odd to me to include in the CMS configuration because the branding API is an LMS thing.  After looking further I realized that we are reusing this cache timeout value in our theming logic -- see https://github.com/edx/edx-platform/pull/11885/files#diff-395f8f21f3f6cb80d7f9c532b06459f5R324.  

I think it would be better to introduce a new set of "Theming" configuration parameters in the settings files and we could have THEME_CACHE_TIMEOUT (or THEME_TEMPLATE_CACHE_TIMEOUT) specifically for theming functionality.